tests/external/downloader/aws/test_aws_report_downloader.py

```diff
@@ -262,6 +263,27 @@ def test_download_file(self):
         out, _ = self.report_downloader.download_file(fake_object)
         self.assertEqual(out, DATA_DIR+'/'+self.fake_customer_name+'/aws/'+fake_object)
 
+    @mock_s3
+    def test_download_file_missing_key(self):
+        fake_object = self.fake.word().lower()
+        conn = boto3.resource('s3', region_name=self.selected_region)
+        conn.create_bucket(Bucket=self.fake_bucket_name)
+        conn.Object(self.fake_bucket_name, fake_object).put(Body='test')
+
+        missing_key = 'missing' + fake_object
+        with self.assertRaises(AWSReportDownloaderError) as error:
+            _, _ = self.report_downloader.download_file(missing_key)
+        expected_err = 'Unable to find {} in S3 Bucket: {}'.format(missing_key, self.fake_bucket_name)
+        self.assertEqual(expected_err, str(error.exception))
+
+    @mock_s3
+    def test_download_file_other_error(self):
+        fake_object = self.fake.word().lower()
+        # No S3 bucket created
+        with self.assertRaises(AWSReportDownloaderError) as error:
+            _, _ = self.report_downloader.download_file(fake_object)
```

Comment:
Don't assign variables if you aren't going to use them. Also, don't use `_` just to avoid pylint complaining about unused-variables.

Let's simplify this down to:

```
        with self.assertRaises(AWSReportDownloaderError) as error:
            self.report_downloader.download_file(fake_object)
```